core/src/main/scala/org/apache/spark/storage/ShuffleBlockFetcherIterator.scala

```diff
@@ -726,6 +736,56 @@ final class ShuffleBlockFetcherIterator(
     }
   }
 
+  // Number of map blocks in a ShuffleBlockChunk
+  private def getShuffleChunkCardinality(blockId: ShuffleBlockChunkId): Int = {
+    pushBasedFetchHelper.getRoaringBitMap(blockId).map(_.getCardinality).getOrElse(0)
+  }
```

Comment:
Should we move this to `PushBasedFetchHelper` since most of the utils belonging to pushShuffle are there?